wrapper/fmriprep_docker.py

```diff
@@ -279,15 +288,8 @@ def get_parser():
     g_dev = parser.add_argument_group(
         'Developer options',
         'Tools for testing and debugging FMRIPREP')
-    g_dev.add_argument('-f', '--patch-fmriprep', metavar='PATH',
-                       type=os.path.abspath,
-                       help='working fmriprep repository')
-    g_dev.add_argument('-n', '--patch-niworkflows', metavar='PATH',
-                       type=os.path.abspath,
-                       help='working niworkflows repository')
-    g_dev.add_argument('-p', '--patch-nipype', metavar='PATH',
-                       type=os.path.abspath,
-                       help='working nipype repository')
+    g_dev.add_argument('--patch', nargs="+", metavar="PACKAGE=PATH", action=ToDict,
```

Comment:
```suggestion
    g_dev.add_argument('-p', '--patch', nargs="+", metavar="PACKAGE=PATH", action=ToDict,
```

Reply:
i'd prefer to leave this explicit tbh, especially since `-p` was used for a specific patch before. though if you feel strongly, we can add it in.

Reply:
I don't feel strongly.